jenkins/linux_c_option_test.sh

```diff
@@ -40,6 +40,7 @@ declare -a arr=(
     "-Denable_raw_logging=ON"
     "-Denable_raw_logging=ON -Dno_logging=ON"
     "-Duse_builtin_httpapi=ON"
+    "-Duse_default_uuid=ON"
```

Comment:
Does setting this need to be added to any documentation we have for regular runs?

Reply:
I added a line in the README.md